src/v2/Apps/Consign/Utils/__tests__/formatCentsToDollars.jest.tsx

```diff
@@ -0,0 +1,12 @@
+import { formatCentsToDollars } from "../formatCentsToDollars"
+
+describe("formatCentsToDollars", () => {
```

Comment:
Nice tests but I'm sorta surprised we don't already have something like this? 🤔

Reply:
I copied this over from the myCollection section in Eigen, but i can follow up!

Reply:
Yup, first use in Force /shrug